lib/src/common.ts

```
@@ -28,7 +28,7 @@ const handleHttpResponse = (res: http.IncomingMessage, resolve: resolver, reject
         try {
             resolve(JSON.parse(body));
         } catch (err) {
-            reject(err);
+            resolve({});
         }
```

This is dangerous because it can mask other errors.
I'd rather:
```
if (body === '') {
    resolve({});
} else {
    try {
            resolve(JSON.parse(body));
        } catch (err) {
            reject(err);
        }
}
```